http-server-netty/src/test/groovy/io/micronaut/http/server/netty/MaxRequestSizeSpec.groovy

```diff
@@ -320,11 +320,12 @@ class MaxRequestSizeSpec extends Specification {
         }
 
         cleanup:
+        responses.forEach(r -> r.release())
         channel.close()
         embeddedServer.close()
     }
 
-    @PendingFeature
+    @Ignore
```

Comment:
This test is broken? or not yet implemented?

Reply:
it has a buffer leak which causes the leak extension to fail even with `@PendingFeature`

Reply:
I see, `@PendingFeature` executes the tests and if the tests fails it considers it green, if it passes it fails the test to tell you that you can remove the annotation.